ccloud-monitoring/docs/consumer-scenarios/increasing-consumer-lag.rst

```diff
@@ -0,0 +1,168 @@
+   The container scaling can be done with the command below:
+
+   .. code-block:: bash
+
+      docker-compose up -d --scale consumer=1 --scale producer=5
+
+   Which will produce the following output:
+
+   .. code-block:: bash
+
+      ccloud-exporter is up-to-date
+      kafka-lag-exporter is up-to-date
+      node-exporter is up-to-date
+      grafana is up-to-date
+      prometheus is up-to-date
+      Stopping and removing ccloud-monitoring_consumer_2 ... done
+      Starting ccloud-monitoring_producer_1              ... done
+      Creating ccloud-monitoring_producer_2              ... done
+      Creating ccloud-monitoring_producer_3              ... done
+      Creating ccloud-monitoring_producer_4              ... done
+      Creating ccloud-monitoring_producer_5              ... done
+      Starting ccloud-monitoring_consumer_1              ... done
+
+Diagnose the problem
+^^^^^^^^^^^^^^^^^^^^
+
+#. Open `Grafana <localhost:3000>`__ and login with the username ``admin`` and password ``password``.
+
+#. Navigate to the ``Consumer Client Metrics`` dashboard. Wait 2 minutes and then observe:
+
+   - ``Rebalance rate`` (``rebalance-rate-per-hour``) has a bump, indicating that the consumer group ``demo-cloud-monitoring-1`` underwent a rebalance, which is to be expected when a consumer leaves the group.
+
+   |Consumer Rebalance Bump|
+
+   - An upward trend in ``Consumer group lag in records``.  ``Consumer group lag in seconds`` will have a less dramatic increase.
+     Both indicate that the producer is creating more messages than the consumer can fetch in a timely manner.
+     These metrics are derived from the ``kafka-lag-exporter`` container.
+     `kafka-lag-exporter <https://github.com/lightbend/kafka-lag-exporter>`__ is a scala open source project that collects data about consumer groups and presents them in a Prometheus scrapable format.
+
+   |Consumer Lag|
+
+   - An increase in ``Fetch request rate`` (``fetch-total``) and ``Fetch size avg`` (``fetch-size-avg``) in the ``Consumer Fetch Metrics`` tab, indicating the consumer is fetching more often and larger batches.
+
+   |Consumer Fetch Increase|
+
+   - All of the graphs in the ``Throughput`` are indicating the consumer is processing more bytes/records.
+
+   |Consumer Throughput Increase|
+
+#. Another view of consumer lag can be found in |ccloud|. Open the UI, navigate to the "Consumers" section and click on the ``demo-cloud-monitoring-1`` consumer group.
+   This page will update periodically, within two minutes you should see a steady increase is the offset lag.
+
+   |Confluent Cloud Consumer Lag|
+
+   This provides a snapshot in time, but it lacks of historical context that the ``Consumer Client Metrics`` dashboard provides.
```

Comment:
```suggestion
   This provides a snapshot in time, but it lacks the historical view that the ``Consumer Client Metrics`` dashboard provides.
```

Reply:
resolved in commit https://github.com/confluentinc/examples/pull/948/commits/9b6e040d6ad2773e51a2e0a49e50b944549ac902